src/Graphics/OpenGLContext/GLSL/glsl_Utils.cpp

```diff
@@ -55,6 +55,13 @@ bool Utils::checkShaderCompileStatus(GLuint obj)
 bool Utils::checkProgramLinkStatus(GLuint obj)
 {
 #ifdef GL_DEBUG
+	checkProgramLinkStatusRequired(obj);
+#endif
+	return true;
+}
+
+bool Utils::checkProgramLinkStatusRequired(GLuint obj)
+{
```

Comment:
What is the purpose of that modification?

Reply:
Well, it adds a ```checkProgramLinkStatusRequired``` method. If that is used instead of ```checkProgramLinkStatus``` then it will check if a GLSL program linked successfully even if the  ```GL_DEBUG``` macro is disabled.

If you are referring to me calling ```checkProgramLinkStatusRequired``` within ```checkProgramLinkStatus``` then or was simply to keep common code.

Reply:
Yep, sorry, it was a stupid question.
I understand what the code does, but I dislike that solution with the new method. Btw, it has an error - there should be
`return checkProgramLinkStatusRequired(obj);`
I corrected your code a little bit - I added `_force` parameter to `checkProgramLinkStatu` - e129d66c22

Reply:
Ok, thanks.